src/pages/bookingsPage/bookingsPage.js

```diff
@@ -4,15 +4,14 @@ import "./bookingsPageStyles.css"
 export default function BookingPage() {
 
   const [bookingsTable, setData] = useState([]);
```

Comment:
variables names can be better, `bookingsTable` should be  `bookingsData` same thing can be done with `setData` variable